### ISSUE TITLE
Transcode pod is not created with subpath 

### ISSUE THREAD

**Issue by OGKevin** [munnerz/kube-plex#49]
When i play a video that requires transcoding, the transcode pod dies immediately with the following logs:

```
container_linux.go:247: starting container process caused "chdir to cwd (\"/transcode/Transcode/Sessions/plex-transcode-4prvaib6gcbct1asr5v9iqce-95126e93-4ec7-46e7-b0ee-5bb2edefb44e\") set in config.json failed: no such file or directory"
```

however, I'm not quite sure why this is happening. When i enter the main plex container i can see the dir:

```
kubectl exec -it plex-kube-plex-65564f564c-b5b7p bash
root@plex-kube-plex:/# ls transcode/Transcode/Sessions/
plex-transcode-4prvaib6gcbct1asr5v9iqce-95126e93-4ec7-46e7-b0ee-5bb2edefb44e
```

any ideas?

**Comment by OGKevin** [comment]
Ahh the PMS pods are not using subpaths while the main plex pod does if you specify it!

https://github.com/munnerz/kube-plex/blob/df3daeafd728c4f0d5c567b2d9bb0565c5cf5e52/main.go#L138-L164

https://github.com/munnerz/kube-plex/blob/df3daeafd728c4f0d5c567b2d9bb0565c5cf5e52/charts/kube-plex/templates/deployment.yaml#L102-L120

I'll see if i can make a PR to fix this :)

**Comment by OGKevin** [comment]
hmm just re-read the k8s doc on subpath now I'm not entirely sure if this is the root cause.

**Comment by OGKevin** [comment]
Yea its due to subpath. Plz let me know if a PR is welcome.

**Comment by gandazgul** [comment]
@OGKevin did you resolve this? I need this feature.

**Comment by OGKevin** [comment]
@gandazgul i could not use subpath, as i needed to rebuild the binary that was spawning the pods and rebuild the image. So i just opted to not use subpath as it was too much of a hassle.

I ended up not being able to use this as the spawend pod that transcode kept dying or was not able to transcode and i just gave up on it. It workes fine w out the transcode pods. If i see i need them again ill prob dig into it. But didn't see the benefit as it works fine with out.

**Comment by gandazgul** [comment]
yeah, I do get some issues with network speed but I'm sure if its the pod or something else. ok. thanks for the reply.
